src/main/java/net/dv8tion/jda/api/requests/restaction/interactions/ReplyCallbackAction.java

```diff
@@ -53,4 +53,17 @@ public interface ReplyCallbackAction extends InteractionCallbackAction<Interacti
     @Nonnull
     @CheckReturnValue
     ReplyCallbackAction setEphemeral(boolean ephemeral);
+
+    /**
+     * Set whether this message should trigger push/desktop notifications to other users.
+     * <br>When a message is silent, it will not trigger push/desktop notifications.
+     *
+     * @param  silent
+     *         True, if this message should not trigger push/desktop notifications
+     *
+     * @return The same reply action, for chaining convenience
+     */
+    @Nonnull
+    @CheckReturnValue
+    ReplyCallbackAction setSilent(boolean silent);
```

Comment:
You can remove all these everywhere except `MessageCreateRequest`, since all these interfaces inherit from it.

Reply:
What do you mean by everywhere ?